drivers/linux/chipsec_km.c

```diff
@@ -560,13 +560,24 @@ int __weak phys_mem_access_prot_allowed(struct file *file,
         return 1;
 }
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,1,12) && defined(ARCH_HAS_VALID_PHYS_ADDR_RANGE) && !defined(__HAVE_ARCH_PAX_OPEN_USERLAND)
+#if defined(__HAVE_ARCH_PAX_OPEN_USERLAND)
```

Comment:
From the previous version it is asking for !defined - is this a valid change?

Reply:
it should be valid if you're using the grsec kernel > 4.14.